### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1

### ISSUE THREAD

**Issue by MagTun** [jay0lee/got-your-back#222]
Full steps to reproduce the issue:
1. Download a Gmail backup as .mbox from https://takeout.google.com/  
2. Run the command  `gyb --email youremail@gmail.com --action restore-mbox --local-folder "c:\my_gmail_backup"`

Expected outcome (what are you trying to do?):
The full .mobx should be restored to my gmail account

Actual outcome (what errors or bad behavior do you see instead?):
Everything runs smoothly until I get this error:

```
Traceback (most recent call last):
  File "gyb.py", line 2132, in <module>
  File "gyb.py", line 1825, in main
  File "fmbox.py", line 27, in get_header
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)


```

Many emails in the `.mobx` are in French so they have some weird characters like `à, é, ù, î, ö, ç...` 

I have check this [issue](https://github.com/jay0lee/got-your-back/issues/125#issuecomment-298003584) but the fix `gyb --action split-mbox` returns this error:

`split-mbox is no longer necessary and is deprecated. Mbox file size should not impact restore performance in this version.`

I have also seen this [issues ](https://github.com/jay0lee/got-your-back/issues/51#issuecomment-171827569) but the fix was added to the master branch 

I have check the last messages send in the `-restored.sqlite` to then check them in `msg-db.sqlite` as recommended [here](https://github.com/jay0lee/got-your-back/issues/51#issuecomment-139922167) but my `msg-db.sqlite` is empty (probably because I got my .mbox from google and bit from `gyb`)

I confirm that 
* I have upgraded to the latest GYB release from https://github.com/jay0lee/got-your-back/releases and I still have this issue.
* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/got-your-back/wiki

gyb --version returns:
```
Got Your Back 1.30
https://git.io/gyb
....@gmail.com
Python 3.7.4 64-bit final
google-api-client 1.7.11
Windows-10-10.0.18362 AMD64
Path: C:\Program Portable\gyb (gmail bkp)
OpenSSL 1.1.1c  28 May 2019

```

**Comment by MagTun** [comment]
I checked the files mentioned in the error and I found this:

```
      if line.startswith(b'%s: ' % header):
        return line[len(header)+2:].decode('ascii')
```

It should probably be changed to:
` return line[len(header)+2:].decode('utf-8')`

**Comment by MagTun** [comment]
I have downloaded a zip of the master branch of your repository and I replaced the two `'ascii'` by `'utf-8'` in `fmbox.py` according to my previous comment. 
Then I run everything with this command:
`python gyb.py --email youremail@gmail.com --action restore-mbox --local-folder "c:\my_gmail_backup"`

This fix seems to have fix the error.  (I least I don't get anymore the error at 8% like I used to get each time I tried - I am now at 10%). At will let you know!
